.styleci.yml

```diff
@@ -0,0 +1,8 @@
+enabled:
+
+disabled:
+  - align_double_arrow
```

Comment:
I would recommend to just use the ``symfony`` preset: https://github.com/doctrine/DoctrinePHPCRBundle/blob/master/.styleci.yml